config/locales/defaults/download_upload/es.yml

```diff
@@ -12,6 +12,7 @@ es:
   upload: Subir
   upload_errors:
     invalid_csv_row_format: El formato de la fila es inválido
+    invalid_file_type: UPDATE ME
```

Comment:
If a field should have a variable, include it even when we're just doing 'UPDATE ME'. So this should be:

```
invalid_file_type: UPDATE ME %{type}
```

And the same for the other translations below